client/my-sites/customer-home/cards/education/mastering-gutenberg/index.jsx

```diff
@@ -0,0 +1,122 @@
+/**
+ * External dependencies
+ */
+import React, { Component, Fragment } from 'react';
+import { connect } from 'react-redux';
+import { localize } from 'i18n-calypso';
+import { Button, Dialog } from '@automattic/components';
+import { isMobile } from '@automattic/viewport';
+
+/**
+ * Internal dependencies
+ */
+import DismissibleCard from 'blocks/dismissible-card';
+import CardHeading from 'components/card-heading';
+import Gridicon from 'components/gridicon';
+import { localizeUrl } from 'lib/i18n-utils';
+import isClassicEditorForced from 'state/selectors/is-classic-editor-forced';
+import { getSelectedSiteId } from 'state/ui/selectors';
+
+/**
+ * Style dependencies
+ */
+import './style.scss';
+
+class MasteringGutenberg extends Component {
+	state = {
+		activeDialog: false,
+	};
+
+	handleDialogClosure = () => {
+		this.setState( {
+			showDialog: false,
+			activeDialog: false,
+		} );
+	};
+
+	handleButtonClick = button => {
+		this.setState( { showDialog: true, activeDialog: button } );
+	};
+
+	render() {
+		const { cannotUseGutenberg, translate } = this.props;
+		const { activeDialog, showDialog } = this.state;
+
+		if ( cannotUseGutenberg ) {
+			return null;
+		}
+
+		let supportLink;
+		let videoId;
+
+		switch ( activeDialog ) {
+			case 'adjustingBlocks':
+				videoId = 'oQNap9YTrWw';
+				supportLink = localizeUrl( 'https://wordpress.com/support/wordpress-editor' );
+				break;
+			case 'customizingWithBlocks':
+				videoId = 'BkQlTuJ4BH8';
+				supportLink = localizeUrl( 'https://wordpress.com/support/wordpress-editor/blocks/' );
+				break;
+		}
+
+		return (
+			<Fragment>
+				<Dialog additionalClassNames="mastering-gutenberg__dialog" isVisible={ showDialog }>
+					<iframe
+						title={ translate( 'Video demonstration' ) }
+						width="560"
+						height="315"
+						src={ `https://www.youtube.com/embed/${ videoId }?rel=0&autoplay=1` }
```

Comment:
Looking back at the free image library card it looks like we ended up updating the support page (adding the video to the top) and reusing the Support Article Dialog. How do folks feel if follow that pattern and update support articles with the new videos. I'm also okay with the showing the videos directly if we update the modal dialog size.

| Free Images | Mastering the Block Editor |
| ------------- | ------------- |
| <img width="1568" alt="Screen Shot 2020-04-07 at 4 10 32 PM" src="https://user-images.githubusercontent.com/1270189/78727883-c1f59900-78ea-11ea-85a7-3e14c8348b6f.png"> | <img width="1554" alt="Screen Shot 2020-04-07 at 4 10 43 PM" src="https://user-images.githubusercontent.com/1270189/78727962-f36e6480-78ea-11ea-95e9-33edda27b8e3.png"> |

If folks are okay with the suggestion @mmtr or @kwight let's update https://wordpress.com/support/wordpress-editor/ to add the first video at the top.

@arunsathiya do you happen to know offhand of a good support page match for [customizing pages and posts](https://www.youtube.com/watch?v=BkQlTuJ4BH8)?

Reply:
I'm finding it a challenge to make the video responsive, although I've made it a bit larger. It's probably better to embed it in the support document, but I'm not sure there's a specific document to match the contents of the video like there is for the Free Photo Library.

Reply:
[This commit](https://github.com/Automattic/wp-calypso/pull/40080/commits/7eea97e01b1d0d83b8349f5e7df7cce698a1c89e) might help you. We used that back when the free photo library card only displayed a youtube video.

Reply:
But yeah, in general I prefer to follow the pattern of showing a support article dialog that contains the video as we are doing for the new checklist task for updating site menus (https://github.com/Automattic/wp-calypso/pull/40844). I'll update the support articles with the videos, so we can use them here.

Reply:
> If folks are okay with the suggestion @mmtr or @kwight let's update https://wordpress.com/support/wordpress-editor/ to add the first video at the top.

Not sure if that's the best article for https://www.youtube.com/watch?v=oQNap9YTrWw. The video already starts with a block inserted but seems that https://wordpress.com/support/wordpress-editor/ tries to provide guidance for more basic stuff (such as inserting a block). 

Actually, I think https://www.youtube.com/watch?v=BkQlTuJ4BH8 is a better fit for https://wordpress.com/support/wordpress-editor/ since it shows how to add blocks, arrange and move them around which matches the content of the support article.

@arunsathiya would you agree?

Reply:
>@arunsathiya would you agree?

Yup, I agree, @mmtr! 

>@arunsathiya do you happen to know offhand of a good support page match for customizing pages and posts?

https://wordpress.com/support/wordpress-editor/ comes closest. It has also been a while since I did WPcom support, and this page has changed a lot since my last visit. :) I also noticed that this YouTube has a mention of how to add images from the media library, but it's not covered within this support document. https://wordpress.com/support/wordpress-editor/blocks/image-block/ does that instead. It's only a small part of the video though, so, I guess we can continue to consider https://wordpress.com/support/wordpress-editor/ as the best match.

Reply:
Thanks @arunsathiya! Actually, after taking a deeper look at https://wordpress.com/support/wordpress-editor/ and the videos, I think both videos could be included in the same article.

https://www.youtube.com/watch?v=BkQlTuJ4BH8 covers [Adding a Block](https://wordpress.com/support/wordpress-editor/#adding-a-block), [Moving Blocks](https://wordpress.com/support/wordpress-editor/#moving-blocks), [Block Navigation](https://wordpress.com/support/wordpress-editor/#block-navigation) and [Removing a Block](https://wordpress.com/support/wordpress-editor/#removing-a-block) while https://www.youtube.com/watch?v=oQNap9YTrWw covers [Configuring a Block](https://wordpress.com/support/wordpress-editor/#configuring-a-block).

Unfortunately, the inline support dialog cannot point to a specific part of an article, so the options we have are:
- Add both videos to https://wordpress.com/support/wordpress-editor. The card should include then one single link to that article.
- Keep showing videos directly without loading any support article.
- Add only https://www.youtube.com/watch?v=BkQlTuJ4BH8 to https://wordpress.com/support/wordpress-editor. The 1st link of the card (_Adjusting setting of blocks_) should open directly. The 2nd link (_Customizing pages and posts with blocks_) will open the support article.

@Automattic/dotcom-manage-design @Automattic/manage do you have any preference?

Reply:
> Unfortunately, the inline support dialog cannot point to a specific part of an article, so the options we have are:

Hmm, can we not load to an anchor point? Seems like something we can update in a separate PR

Reply:
> Unfortunately, the inline support dialog cannot point to a specific part of an article, so the options we have are:

I've added this in https://github.com/Automattic/wp-calypso/pull/40937

Reply:
I also added the videos in https://wordpress.com/support/wordpress-editor/#blocks and https://wordpress.com/support/wordpress-editor/#configuring-a-block